main.js

```diff
@@ -3,6 +3,13 @@ var roleUpgrader = require('role.upgrader');
 var harvesterCount = 20;
```

Comment:
we can start with less number as all the resource is been hogged by the builder. May be start with 5.